app/services/reaction_points_calculator.rb

```diff
@@ -0,0 +1,76 @@
+class ReactionPointsCalculator
```

Comment:
I believe that in recent months we haven't been enforcing the convention of interfacing all service classes with the `.call` static method, while also naming them as actions ("verbs" of sorts), i.e. something along the lines of `CalculateReactionPoints.call(...)`

**Non-blocking question:** How do you feel about this convention? Is it worth revisiting this as a conversation amongst all of the engineers?

I'm asking because it does feel like there are some benefits to it, but also don't want to push hard or block this PR because of this 🙂

Reply:
> How do you feel about this convention? Is it worth revisiting this as a conversation amongst all of the engineers?

:thumbsup: I either was not aware of this convention or had forgotten about it and it's easy enough to conform this PR towards the convention. 

My off-the-top-of-mind reaction is that I'm not, like, _super_ fond of having a specific convention for service object call signatures. I'm probably more tuned in to the drawbacks, so when/if we convene a conversation with other engineers, I would be particularly interested to hear about the advantages.